webapp/channels/src/components/admin_console/billing/billing_subscriptions/to_yearly_nudge_banner.tsx

```diff
@@ -182,6 +188,10 @@ const ToYearlyNudgeBanner = () => {
         return null;
     }
 
+    if (subscription?.has_payment_method && (subscription.billing_type === CloudBillingTypes.INTERNAL || subscription.billing_type === CloudBillingTypes.LICENSED)) {
```

Comment:
I believe some (if not all) the `internal` ones don't have a payment method. Do we need to check here for payment method? @nickmisasi

Reply:
Correct - i think I might have confused things with my comment on the thread in MM. You can probably just leave out the has_payment_method check entirely here, since paid monthly should still see the banners?